task3/task3.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"github.com/gorilla/mux"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"net/url"
+	"time"
+	"unicode/utf8"
+)
+
+func handleGET(w http.ResponseWriter, r *http.Request) {
+	filename := "index.html"
+	body, err := ioutil.ReadFile(filename)
+	if err != nil {
+		log.Println(err)
+		w.WriteHeader(http.StatusInternalServerError)
+		w.Write([]byte(err.Error()))
```

Comment:
That will lead to error( AFAIR).
Headers can't be sent after body. On error code wouldn't exit here.
```go
w.WriteHeader(http.StatusInternalServerError)
w.Write([]byte(err.Error()))
w.WriteHeader(http.StatusOK)
w.Write([]byte(body))
```